src/sql/base/browser/ui/table/plugins/buttonColumn.plugin.ts

```diff
@@ -28,8 +30,9 @@ export class ButtonColumn<T extends Slick.SlickData> implements Slick.Plugin<T>
 	private _grid: Slick.Grid<T>;
 	private _onClick = new Emitter<ButtonClickEventArgs<T>>();
 	public onClick = this._onClick.event;
+	public index: number;
 
-	constructor(private options: ButtonColumnOptions) {
+	constructor(private options: ButtonColumnOptions, columnIndex?: number) {
```

Comment:
as we chatted previously, you shouldn't need to pass in the column index, please refer to isCurrentColumn method in this file